Oracle/diagnosis.sql

```diff
@@ -131,7 +153,31 @@ insert into originfact --CDM 3.1 addition
 execute immediate 'create index originfact_idx on originfact (patient_num, encounter_num, provider_id, concept_cd, start_date)';
 GATHER_TABLE_STATS('ORIGINFACT');
 
-insert into diagnosis (patid,	encounterid, enc_type, admit_date, providerid, dx, dx_type, dx_source, dx_origin, pdx)
+insert into poafact
+	select patient_num, encounter_num, provider_id, concept_cd, start_date, 'Y' poasource, 'Yes' rawpoasource, dxsource.c_fullname
+	from i2b2fact factline
+    inner join ENCOUNTER enc on enc.patid = factline.patient_num and enc.encounterid = factline.encounter_Num
+    inner join pcornet_diag dxsource on factline.modifier_cd = dxsource.c_basecode
+	and dxsource.c_fullname like '\PCORI_MOD\DX_ORIGIN\BI\DX|BILL:POA\%'
+
+	union all
```

Comment:
The union combines POA (present on admission) facts from observation_fact and supplemental_fact.  The observation facts are always true and the distinction between no, unknown, no information, etc. is lost.  Supplemental facts fills in the gaps.  

I don't cotton to the split approach.  Ultimately, the supplemental facts should be added to observation facts with the code or name in the tval_char field.

I need to add this comment to the code.